vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/model/implementation/Serializer.java

```diff
@@ -0,0 +1,260 @@
+	 * @return String representation corresponding to a given
+	 *         {@link UniversalVariable}.
+	 */
+	public static String getString(UniversalVariable universalVariable) {
+		return universalIdentifier + universalVariable.getName();
+	}
+
+	/**
+	 * Creates a String representation of a given {@link NamedNull}.
+	 * 
+	 * @see <"https://github.com/knowsys/vlog4j/wiki"> for wiki.
+	 * @param namedNull a {@link NamedNull}
+	 * @return String representation corresponding to a given {@link NamedNull}.
+	 */
+	public static String getString(NamedNull namedNull) {
+		return namedNullIdentifier + namedNull.getName();
+	}
+
+	/**
+	 * Creates a String representation of a given {@link Predicate}.
+	 * 
+	 * @see <"https://github.com/knowsys/vlog4j/wiki"> for wiki.
+	 * @param predicate a {@link Predicate}
+	 * @return String representation corresponding to a given {@link Predicate}.
+	 */
+	public static String getString(Predicate predicate) {
+		return predicate.getName() + openParenthesis + predicate.getArity() + closingParenthesis;
+	}
+
+	/**
+	 * Creates a String representation of a given {@link DataSourceDeclaration}.
+	 * 
+	 * @see <"https://github.com/knowsys/vlog4j/wiki"> for wiki.
+	 * @param dataSourceDeclaration a {@link DataSourceDeclaration}
+	 * @return String representation corresponding to a given
+	 *         {@link DataSourceDeclaration}.
+	 */
+	public static String getString(DataSourceDeclaration dataSourceDeclaration) {
+		return "@source " + dataSourceDeclaration.getPredicate().getName() + openParenthesis
+				+ dataSourceDeclaration.getPredicate().getArity() + closingParenthesis + ": "
+				+ dataSourceDeclaration.getDataSource().getSyntacticRepresentation();
+	}
+
+	/**
+	 * Creates a String representation of a given {@link Conjunction}.
+	 * 
+	 * @see <"https://github.com/knowsys/vlog4j/wiki"> for wiki.
+	 * @param conjunction a {@link Conjunction}
+	 * @return String representation corresponding to a given {@link Conjunction}.
+	 */
+	public static String getString(Conjunction<? extends Literal> conjunction) {
+		final StringBuilder stringBuilder = new StringBuilder();
+		boolean first = true;
+		for (final Literal literal : conjunction.getLiterals()) {
+			if (first) {
+				first = false;
+			} else {
+				stringBuilder.append(comma + " ");
+			}
+			stringBuilder.append(getString(literal));
+		}
+		return stringBuilder.toString();
+	}
+
+	/**
+	 * Creates a String representation corresponding to the name of a given
+	 * {@link LanguageStringConstant}.
+	 * 
+	 * @see <"https://github.com/knowsys/vlog4j/wiki"> for wiki.
+	 * @param languageStringConstant a {@link LanguageStringConstant}
+	 * @return String representation corresponding to the name of a given
+	 *         {@link LanguageStringConstant}.
+	 */
+	public static String getConstantName(LanguageStringConstant languageStringConstant) {
+		return "\"" + languageStringConstant.getString().replace("\\", "\\\\").replace("\"", "\\\"") + "\"@"
```

Comment:
We could use have a method (for example, `private String addQuotations(String string)`) for `return "\""+sting+"\""`;